### DISCUSSION TITLE
ffmpeg or ffmpeg4.4

### DISCUSSION THREAD

**Discussion by gearhead** [mikebrady/shairport-sync#1608] [General]
I have been building and using this for a while. I recently wanted to try out the 4.1 version. I was able to get it to build and am starting to check it out. One question I have, though, is which ffmpeg is required? I am on arch arm and the 'current' ffmpeg is installed and required by other packages I am using is `ffmpeg 2:5.1.2-1` I was looking in the AURs and ran across this:
[https://aur.archlinux.org/packages/shairport-sync-git](url) and they claim we need ffmpeg4 which is currently at `ffmpeg4.4 4.4.3-1`. Can you shed any light on this? What exactly is required? I'm not actually building the git, but the release of 4.1.1. Do you know if I can uninstall ffmpeg and just install ffmpeg4?

**Comment by mikebrady** [comment]
Thanks for the post. Shairport Sync requires an ffmpeg that includes support for decoding Planar Floating Point -- `fltp` -- AAC material. There's a short discussion of it [here](https://github.com/mikebrady/shairport-sync/blob/master/TROUBLESHOOTING.md#aac-decoder-issues-airplay-2-only) in the [TROUBLESHOOTINGmd](https://github.com/mikebrady/shairport-sync/blob/master/TROUBLESHOOTING.md) guide.
